packages/eslint-config-custom/package.json

```diff
@@ -9,6 +9,7 @@
     },
     "dependencies": {
         "@typescript-eslint/eslint-plugin": "5.61.0",
+        "eslint": "^8.45.0",
```

Comment:
eslint dependency instead of devdependency?

Reply:
yeah, and i think all of the others too. Will fix it.